composer.lock

```diff
@@ -5378,16 +5378,16 @@
         },
         {
             "name": "symfony/mailer",
-            "version": "v4.4.38",
+            "version": "v4.4.41",
```

Comment:
Cool to see the symfony mailer was already in the project. 
Did you intentionally not add it to the composer.json file?

Reply:
It is currently also in the composer.json file.